packages/webview_flutter/webview_flutter/example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>8.0</string>
+  <string>9.0</string>
```

Comment:
The webview_flutter/webview_flutter changes should be reverted.

Reply:
Done.